mobile/ios/ios-device.ts

```diff
@@ -136,6 +136,12 @@ export class IOSDevice implements Mobile.IIOSDevice {
 		}).future<void>()();
 	}
 
+    public debug(packageFile: string, packageName: string): IFuture<void> {
+        return (() => {
+            // TODO: add implementation
```

Comment:
Consider giving a helpful error message in here: 
```this.$errors.fail({formatStr:"this will come in a future version", suppressCommandHelp: true})``` or similar.